static/app/components/onboardingWizard/newSidebar.tsx

```diff
@@ -43,6 +43,24 @@ const INITIAL_MARK_COMPLETE_TIMEOUT = 600;
  */
 const COMPLETION_SEEN_TIMEOUT = 800;
 
+const orderedGettingStartedTasks = [
+  OnboardingTaskKey.FIRST_PROJECT,
+  OnboardingTaskKey.FIRST_EVENT,
+  OnboardingTaskKey.INVITE_MEMBER,
```

Comment:
This one I don't quite understand: why are users expected to invite a member directly after their first event?